Source/WebCore/Modules/indexeddb/shared/IDBResourceIdentifier.h

```diff
@@ -44,6 +44,9 @@ class IDBConnectionToClient;
 using IDBConnectionIdentifier = ProcessIdentifier;
 struct IDBResourceIdentifierHashTraits;
 
+enum class IDBResourceIdentifierResourceNumberType { };
+using IDBResourceIdentifierResourceNumberID = AtomicObjectIdentifier<IDBResourceIdentifierResourceNumberType>;
```

Comment:
`Number` and `ID` seem to mean the same thing here; we could just call this `IDBResourceObjectIdentifier`